facilitator/src/bin/facilitator.rs

```diff
@@ -1343,22 +1343,6 @@ where
         batch_signing_key: batch_signing_key_from_arg(sub_matches)?,
     };
 
-    // We created the bucket to which we write copies of our validation
-    // shares, so it is simply provided by argument.
-    let mut own_validation_transport = SignableTransport {
```

Comment:
Besides deleting the handling of these arguments, we should also be able to remove them from the `intake-batch`, `aggregate`, `intake-batch-worker` and `aggregate-worker` `SubCommand`s defined in `main()`. Additionally, we should be able to remove some entries from the `intake_batch_config_map` and `aggregate_config_map` resources in `terraform/modules/kubernetes/kubernetes.tf`, as well as the `--own-validation-input` argument passed to `workflow-manager`.